packages/expo-analytics-amplitude/src/ExpoAmplitude.web.ts

```diff
@@ -1,5 +1,31 @@
+import amplitude from 'amplitude-js'
```

Comment:
Could you please prettify the file? Our Prettier configuration expects `;` ending all the statements. 🙂